pkg/services/ssosettings/api/api.go

```diff
@@ -99,6 +99,20 @@ func (api *Api) updateProviderSettings(c *contextmodel.ReqContext) response.Resp
 	return response.JSON(http.StatusNoContent, nil)
 }
 
+// swagger:route DELETE /ssoSettings/{key} sso_settings removeProviderSettings
+//
+// # Remove SSO Settings
+//
+// # Remove an SSO Settings entry by Key
+//
+// If you have Fine-grained access control enabled then
```

Comment:
Unless a new decision came up, I think we are trying to avoid references to `Fine-grained access control` and prefer `role base access control`